Sources/OktaLogger/LoggerCore/OktaLoggerLogLevel.swift

```diff
@@ -49,6 +50,8 @@ extension OktaLoggerLogLevel {
             return "🛑"
         case .warning:
             return "⚠️"
+        case .analytics:
+            return "🧐"
```

Comment:
do we want to use this icon or some other icon indicating tracking analytics ?

Reply:
Maybe 📊?

Reply:
📈